lib/sync/sync-server.js

```diff
@@ -361,7 +358,8 @@ module.exports = {
   setClients: setClients,
   api: {
     init: init,
-    start: start,
+    // Memoize the start function so it will only get called at most once
+    start: async.memoize(start),
```

Comment:
are we sure the function will be only called once, even if the second call happens before the first call is completed? It is not that obvious from the description of documentation.